apps/shopify/src/skuResolvers.js

```diff
@@ -177,6 +182,26 @@ export const makeCollectionSearchResolver = async (sdk) => {
   return (search) => pagination.fetchNext(search);
 };
 
+export const filterValidIds = (skus, skuType) => {
```

Comment:
This filters AND converts the ids. The name of the method is not quite accurate anymore